### PR TITLE
added footer bar [skip ci]

### PR DESCRIPTION
**Pull request by rcoffie**
footer-bar

### CODE REVIEW

File: themes/qgis/layouts/partials/footer.html [hunk 1]
```diff
@@ -1,21 +1,69 @@
...
+<footer class="footer has-background-dark has-text-white">
+    <section class="section">
+        <div class="container">
+            <div class="columns has-text-justified">
+                <div class="column">
+                    <ul class="block-list is-small is-outlined is-success is-centered mb-4">
+                        <li class="is-size-5">Download</li>
+                        <li><a href="http://" class="has-text-white">Latest QGIS</a></li>
+                        <li><a href="http://" class="has-text-white">QGIS Latest Releases</a></li>
+                        <li><a href="http://" class="has-text-white">Releases Notes</a></li>
```
Comment (timlinux) [inline themes/qgis/layouts/partials/footer.html:10]: Release Notes

File: themes/qgis/layouts/partials/footer.html [hunk 2]
```diff
@@ -1,21 +1,69 @@
...
+                </div>
+                <div class="column">
+                    <ul class="block-list is-small is-outlined is-success is-centered mb-4">
+                        <li class="is-size-5">About</li>
+                        <li><a href="http://" class="has-text-white">License</a></li>
+                        <li><a href="http://" class="has-text-white">Logo & Trademark</a></li>
+                        <li><a href="http://" class="has-text-white">Privacy Policy</a></li>
+                    </ul>
+                    <ul class="block-list is-small is-outlined is-success is-centered">
+                        <li class="is-size-5"> Organization</li>
```
Comment (timlinux) [inline themes/qgis/layouts/partials/footer.html:27]: No space

File: themes/qgis/layouts/partials/footer.html [hunk 3]
```diff
@@ -1,21 +1,69 @@
...
+                        <li><a href="http://" class="has-text-white">People</a></li>
+                        <li><a href="http://" class="has-text-white"> Jobs</a></li>
```
Comment (timlinux) [inline themes/qgis/layouts/partials/footer.html:29]: No space

File: themes/qgis/layouts/partials/footer.html [hunk 4]
```diff
@@ -1,21 +1,69 @@
...
+                    </ul>
+                </div>
+                <div class="column">
+                    <ul class="block-list is-small is-outlined is-success is-centered">
+                        <li class="is-size-5"> Get Involved</li>
+                        <li><a href="http://" class="has-text-white">Write Documentation</a></li>
+                        <li><a href="http://" class="has-text-white">Translation</a></li>
+                        <li><a href="http://" class="has-text-white">Development</a></li>
+                        <li><a href="http://" class="has-text-white">Tests </a></li>
```
Comment (timlinux) [inline themes/qgis/layouts/partials/footer.html:38]: no space

**Review by timlinux — CHANGES_REQUESTED** [review]
Just fix the small comments here please then I will merge

**Comment by rcoffie** [comment]
Overviews 
- spaces removed 
- Release Notes fixed 
- -added the border left to the Follow QGIS secdtion 
follow-section-border

**Comment by timlinux** [comment]
Lgtm

**Pull request merged**